styles/standard.mapcss

```diff
@@ -816,10 +816,11 @@ node|z14-[railway=tram_stop]
 /*****************/
 /* owner changes */
 /*****************/
-node|z11-[railway=owner_change]
+node|z11-[railway=border],
```

Comment:
Why do you use one icon for both border and owner change? You created different icons, so they should be used.